src/main/java/mate/academy/rickandmorty/dto/external/CharacterResponseDataDto.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.rickandmorty.dto.external;
+
+import java.util.List;
+
+public record CharacterResponseDataDto(List<CharacterResponseDto> result) {
+}
```

Comment:
lets use single code style at the project
```suggestion
public record CharacterResponseDataDto(
            List<CharacterResponseDto> result
) {
}
```